torch/lib/c10d/reducer.cpp

```diff
@@ -1465,6 +1466,29 @@ void Reducer::ensure_prior_reduction_finished() {
   }
 }
 
+void Reducer::set_construction_logging_data(
+  const std::string& backend,
+  const std::string& module_name,
+  const std::vector<int>& device_ids,
+  int output_device,
```

Comment:
In the unit test, output_device is expected to be -1. Probably add some comments to explain when should -1 be used?
Otherwise, I have to look into the implementation details (`dist._set_construction_logging_data` in distributed.py)

Reply:
sounds good!